botocore/handlers.py

```diff
@@ -734,6 +735,17 @@ def convert_body_to_file_like_object(params, **kwargs):
             params['Body'] = six.BytesIO(params['Body'])
 
 
+def fix_route53_max_items(params, model, **kwargs):
+    # Converts the MaxItems param from an integer to a string.
+    # Route53 has defined their MaxItems field for pagination as a string.
+    # This causes issues when the cli automatically converts the page size
+    # to an integer as it will fail validation.
+    if 'MaxItems' in params:
+        if (isinstance(params['MaxItems'], six.integer_types) and
+            isinstance(model.input_shape.members['MaxItems'], StringShape)):
```

Comment:
This should be indented over to not share the same level as the next logical line. If that makes things too long you can also flatten all of it out with guard clauses.